components/Frame/ProlongBox.js

```diff
@@ -42,13 +42,14 @@ const ProlongBox = ({ t, prolongBeforeDate, membership, router }) => {
   const [colorScheme] = useColorContext()
 
   if (
-    router.pathname === '/pledge' ||
-    router.pathname === '/cancel' ||
-    router.pathname === '/meta' ||
+    router.pathname === '/angebote' ||
+    router.pathname === '/abgang' ||
+    router.asPath === '/meta' ||
```

Comment:
Let's remove this line completely, before we had the cockpit meta was a special page used for prolonging (December 18 / January 19). Since meta is now just a regular section we don't need to treat it specially anymore.

Reply:
should we keep cockpit in, though?

Reply:
`abgang` and `angebote` I understand, but `cockpit` is kinda the odd one out, here. (then we might want to include stuff like `about`, too. No?)

Reply:
jup cockpit should stay excluded: it has it's own prolong cta when the membership due

Reply:
the reason the cockpit is excluded is because it can be used in mailings as a landing page for deciding whether one should prolong and has it's own CTA for prolong. Unlike about.

Reply:
the reason the cockpit is excluded is because it can be used in mailings as a landing page for deciding whether one should prolong and has it's own CTA for prolong. Unlike about.